modules/va_forms/app/controllers/va_forms/v0/forms_controller.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module VaForms
+  module V0
+    class FormsController < ApplicationController
+      skip_before_action(:authenticate)
+
+      def index
+        forms = Form.all
```

Comment:
I'm guessing we are going to replace the .all with a search query that defaults to all like we talked about or is that for a different PR?

Reply:
My thought was that we would add that as another PR after we get a little feedback from users